### DISCUSSION TITLE
Fields shown in Quick Details Pop up

### DISCUSSION THREAD

**Discussion by Cattrah** [YetiForceCompany/YetiForceCRM#16023] [How to] [answered]
How do you manage what fields are shown in these detail tooltip popovers when you hover on certain links? They appear for many of the modules, but changing the field configuration to "Summary View" or "Visible in preview header" doesn't seem to change what fields appear in these boxes.

![image](https://user-images.githubusercontent.com/101741600/166254787-92f65e14-218b-40c2-b205-244951e680b6.png)

**Comment by vovpff** [comment]
I think this is hardcoded into calendar template. U need to modify original files to change something

**Reply by Cattrah** [reply]
ok. I am comfortable doing that, do you know what directory the file(s) might be in or what it might be named?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
https://github.com/YetiForceCompany/YetiForceCRM/blob/1cc2f25ac58141198437bda93cd9e12513bacefb/modules/Calendar/models/RecordPopover.php#L33-L40

**Reply by Cattrah** [reply]
Thank you! I added the fields I wanted to the getFieldsIcon() function and they showed right up! Cheers!
